python/hopsworks/job.py

```diff
@@ -47,6 +50,11 @@ def __init__(
         self._creator = creator
         self._executions = executions
         self._project_id = project_id
+        self._job_schedule = (
+            js.JobSchedule.from_response_json(job_schedule)
+            if job_schedule
```

Comment:
```suggestion
            if job_schedule is not None
```

Reply:
In Python `if job_schedule` will return true if the object is not None or empty string.